docs/getting-started/learn-the-api.md

```diff
@@ -0,0 +1,137 @@
+---
+order: 60
+---
+
+# Learn the API
+
+Now that we've created an host application, loaded a few modules and registered routes and navigation items, let's dive into the APIs that are available with this shell.
+
+!!!info
+For an exhaustive list of `@squide` API, have a look at the [References](/references#api) section.
+!!!
+
+## Logging
+
+`@squide` comes with a native logging feature integrated with the [Runtime](/references/runtime/runtime-class.md) class and the [useLogger](/references/runtime/useLogger.md) hook.
+
+First, register your own custom logger by implementing the [Logger](/references/logging/Logger.md) interface or with `@squide` native [ConsoleLogger](/references/logging/ConsoleLogger):
+
+```ts host/src/bootstrap.tsx
+import { Runtime, ConsoleLogger, type LogLevel } from "@squide/react-router";
+
+const runtime = new Runtime({
+    loggers: [new ConsoleLogger(LogLevel.debug)]
+});
+```
+
+Then, log entries from anywhere any part of your federated application with the [useLogger](/references/runtime/useLogger.md) hook:
```

Comment:
The section `from anywhere any part of` doesn't sound right

Reply:
Good catch, thanks, updated.